frontend/src/main/components/Commons/CommonsOverview.js

```diff
@@ -14,16 +16,29 @@ export default function CommonsOverview({ commons, currentUser }) {
     const date2 = new Date();
     const Difference_In_Time = Math.abs(date2.getTime() - date1.getTime());
     const days = Math.ceil(Difference_In_Time / (1000 * 3600 * 24));
+
+    const { data: players } =
+    useBackend(
+       // Stryker disable next-line all : don't test internal caching of React Query
+      [`/api/usercommons/commons/all?commonsId=${commons.id}`],
+      {  // Stryker disable next-line all : GET is the default, so changing this to "" doesn't introduce a bug
+        method: "GET",
+        url: "/api/usercommons/commons/all",
+        params: {
+          commonsId: commons.id
+        }
+      }
+    );
+    
```

Comment:
This is far more information than you need, and it isn't really what you want.

This PR shows how you can implement an endpoint very simply to get what you want directly rather than having the frontend do way too much work, and get way more information than you need (which is inefficient for both the server and the client.)

https://github.com/ucsb-cs156-f22/f22-7pm-happycows/pull/69

Please consider merging that PR into yours, then refactoring to use that endpoint instead.